ecosystem/typescript/sdk/src/indexer/generated/operations.ts

```diff
@@ -38,16 +40,17 @@ export type GetAccountTransactionsCountQueryVariables = Types.Exact<{
 }>;
 
 
-export type GetAccountTransactionsCountQuery = { __typename?: 'query_root', move_resources_aggregate: { __typename?: 'move_resources_aggregate', aggregate?: { __typename?: 'move_resources_aggregate_fields', count: number } | null } };
+export type GetAccountTransactionsCountQuery = { __typename?: 'query_root', account_transactions_aggregate: { __typename?: 'account_transactions_aggregate', aggregate?: { __typename?: 'account_transactions_aggregate_fields', count: number } | null } };
```

Comment:
For SDK V2, let's hopefully clean this up so you don't have to access the sub type `account_transactions_aggregate`.

We should continue doing it this way for SDK v1, because all of them do this

Reply:
types are generated from Hasura schema, so it depends on how it is defined there